static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -0,0 +1,21 @@
+# How to update your pipeline by editing dvc files
```

Comment:
- DVC-files
- simpler name - `Updating pipelines`. Especially considering that you also mention `dvc move`.
- update sidebar.json to connect this new section
- remove .share-data.swp from the PR